tests/test_data.py

```diff
@@ -126,6 +123,7 @@ def test_dataset_creation(self):
         """Load a dataset."""
         dallinger.data.Data(self.data_path)
 
+    @pytest.mark.skip
```

Comment:
Are these skipped because of the `odo` issue (which should be resolved) or something else?

Reply:
This seems to be a separate odo issue sadly. It seems that it uses inspect.getargspec, which has been deprecated for a while and now removed.

Reply:
Ugh, I really think we should bite the bullet and get rid of it while we're making a breaking release. Thankfully we should already have tests for all the functionality that uses `odo`.